plugins/api/discovery/src/test/java/org/apache/cloudstack/discovery/ApiDiscoveryTest.java

```diff
@@ -16,10 +16,27 @@
 // under the License.
 package org.apache.cloudstack.discovery;
 
-import com.cloud.user.User;
-import com.cloud.user.UserVO;
-import com.cloud.utils.component.PluggableService;
+import static org.junit.Assert.assertEquals;
```

Comment:
can you undo the import reorganisation, please?

Reply:
this will be hard with my IDE settings and my preference of alphabetical imports. I'm absolutely if you want to change this and push into PR branch